README.md

```diff
@@ -20,14 +20,25 @@ your installation.  For example ``export QUARTUSPATH="/home/miner/altera/18.1/qu
 This line should be added to your ``~/.profile`` file or another file that is sourced whenever a
 shell is launched.  Don't forget to source the file after editing it.
 
-Solo Mining
------------
+Local Solo Mining
+-----------------
+
+Local Digibyte node is required.
 
 Install and start a full node via <https://github.com/digibyte/digibyte>.
 
 * A python interpreter is required and pip is recommended - ``apt install python python-pip`` (Python 3 should also work, but most testing has been done in Python 2).
 * Python modules base58 and requests - ``pip install base58 requests``
 
+Stratum Pool Mining
+-------------------
+
+Local Digibyte node is *not* required.
+
+* ``src/pool/solo/stratum_proxy.py`` included for stratum-based mining, it should be started instead of ``pool.py``
+* Pool auth user need to be specified at ``src/miner/config.tcl`` and ``config_mode`` set to ``stratum`` as well as ``config_port``. Sample configuration file ``config.tcl.sample.stratum`` is included
```

Comment:
Thank you so much for reviewing this pull request!

It was idea in the mind that exactly miner itself should be authorized against pool, as some 'classic' schema when ASIC or GPU rig is configured individually and you can specify separate worker name for each device for monitoring purposes.

In case if Auth moved to the stratum proxy itself - only proxy itself will be able to auth and it will be the single connection and only mining session.

Absolutely another reason  - this VarDiff from pool.  For the present moment start difficulty from the pool equal to 1(one), it is quite enough for Cyclone V as kind of low entry mining device.

But in case of Arria V (I do not have one, but it is supported as well according to the code) diff 1 may be too low and shares will be sent to a pool too often. For this reason pool will automatically adjust difficulty to higher value to keep balance between work changes and network traffic.

So if someone use multiple devices with the same host PC - they all be handled in the correct way.
Even you will add another high perfomance devices support later.

And also for this reason there are individual connections to a pool for an each mining client, but not just single pool connection.  We do not want make local proxy too complex and include any difficulty estimation and jobs scheduling for an each device - pool can handle this as well much better per worker basis.

What do you think about this?

Reply:
Fair points.  Multiple connections make sense.  But in terms of config, I feel like the (host, port, user, password) info should all be given to a single piece of the chain, rather than host+port given to one piece, and user+password given to a different piece.

Reply:
Done. Moved credentials to stratum.py agruments.